auth/lib/utils/lock_screen_settings.dart

```diff
@@ -38,6 +41,23 @@ class LockScreenSettings {
 
     ///Workaround for privacyScreen not working when app is killed and opened.
     await setHideAppContent(getShouldHideAppContent());
+
+    await setHasMigratedLockScreenChanges();
+  }
+
+  Future<void> setHasMigratedLockScreenChanges() async {
+    if (_preferences.getBool(keyHasMigratedLockScreenChanges) == null ||
```

Comment:
Can change this to just `if (_preferences.getBool(keyHasMigratedLockScreenChanges) == null`.
I couldn't understand why we need to check `getUsAppLockSet` here.